pkg/server/server_test.go

```diff
@@ -1897,3 +1954,31 @@ func TestWriteAuthorizationModelWithSchema12(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestIsExperimentallyEnabled(t *testing.T) {
+	someExperimentalFlag := ExperimentalFeatureFlag("some-experimental-feature-to-enable")
+
+	server := Server{}
```

Comment:
Better yet, please use the options builder pattern, which is what the public consumer of the `server` package would be using. Otherwise there are no guarantees this invariat remains true over time.

e.g.

```go
t.Run("returns_false_if_experimentals_empty", func(t *testing.T) {
    s := MustNewServerWithOpts()

    require.False(t, server.IsExperimentallyEnabled(someExperimentalFlag))
})

t.Run("returns_true_if_experimentals_matches", func(t *testing.T) {
    s := MustNewServerWithOpts(
        WithExperimentals(someExperimentFlag),
    )

    require.True(t, server.IsExperimentallyEnabled(someExperimentalFlag))
})
```

etc...